nerfstudio/model_components/losses.py

```diff
@@ -502,3 +502,29 @@ def tv_loss(grids: TensorType["grids", "feature_dim", "row", "column"]) -> Tenso
     h_tv = torch.pow((grids[:, :, 1:, :] - grids[:, :, :-1, :]), 2).sum()
     w_tv = torch.pow((grids[:, :, :, 1:] - grids[:, :, :, :-1]), 2).sum()
     return 2 * (h_tv / h_tv_count + w_tv / w_tv_count) / number_of_grids
+
+
+class GradientScaler(torch.autograd.Function):  # typing: ignore, pylint: disable=abstract-method
+    """
+    Scale gradients by the ray distance to the pixel
+    as suggested in `Radiance Field Gradient Scaling for Unbiased Near-Camera Training` paper
+
+    Note, the scaling is applied on the interval of [0, 1] along the ray!
```

Comment:
Add example of how to call in docstring. More info [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#:~:text=Example%3A%0A%20%20%20%20Examples%20can%20be%20given%20using%20either%20the%20%60%60Example%60%60%20or%20%60%60Examples%60%60%0A%20%20%20%20sections.%20Sections%20support%20any%20reStructuredText%20formatting%2C%20including%0A%20%20%20%20literal%20blocks%3A%3A%0A%0A%20%20%20%20%20%20%20%20%24%20python%20example_google.py)